apps/barista-design-system/src/components/content-link/content-link.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, Input, HostListener, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+/**
+ * The ba-content link component is used because we need to dynamically
+ * instanciate a router link and this is not possible, because it is a directive.
+ * So we create a component that handles the navigation.
+ */
+@Component({
+  selector: 'a[contentlink]',
+  template: '{{linkValue}}',
```

Comment:
please use an `ng-content` instead of the value content of a link can be more than text